.github/workflows/remake_rst_files.yml

```diff
@@ -6,18 +6,21 @@ on:
 
 jobs:
     remake-rst-files:
-        runs-on: ubuntu-latest
+        runs-on: ubuntu-20.04
```

Comment:
Seems unnecessary to set the ubuntu version away from latest? Should then run tests on multiple versions?

Reply:
I'm pinning the version so that we can be sure what our environment is. I don't want the same command to suddenly start creating new images.

Reply:
To be fair, I  think the image might end up being slightly different anyway, because I do not pin the version on thing I install via `apt-get`.

Reply:
Maybe, once the issue is fixed, we remove this?

Reply:
Maybe, once the issue is fixed, we remove this?

Reply:
Nah I mean I want that guarantee irrespective of the issue. I guess I'm not clear on the benefit of using latest.